apps/wallet/src/ui/app/components/transactions-card/index.tsx

```diff
@@ -47,7 +47,7 @@ function TransactionCard({ txn }: { txn: TxResultState }) {
     const transferType =
         txn.kind === 'Call' ? 'Call' : txn.isSender ? 'Sent' : 'Received';
 
-    const amount = txn?.balance || txn?.amount || txn?.txGas || 0;
+    const amount = txn?.amount || txn?.txGas || 0;
```

Comment:
Why this removal?

Reply:
`txn.balance` seems to refer to the balance of the sender.